### PR TITLE
refine security and signature descriptions

### PR DESCRIPTION
**Pull request by zhangyongjing**
1.  move Figure 4 to Annex B.
2.  rewording and simplify some text for clarity
3.  add some more definions and references for [Signing Block] and [Provisioning Profile]
4.  remove redundant and unused signature IDs values
5.  add an editor's note for 'multi-package' signature
6.  other editorials and typos

@espinr @MichaelWangzitao

***
<a href="https://pr-preview.s3.amazonaws.com/zhangyongjing/miniapp-packaging/pull/23.html" title="Last updated on May 25, 2021, 6:14 AM UTC (6e28214)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/miniapp-packaging/23/0d77d8d...zhangyongjing:6e28214.html" title="Last updated on May 25, 2021, 6:14 AM UTC (6e28214)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -774,8 +764,8 @@ <h6 id="sec-provisioning-profile-id">Provisioning Profile IDs</h6>
...
+            <p>This signature scheme is based on the PKCS #7 Cryptographic Message Syntax [[?rfc2315]]. It MAY cover different stages of packaging signature described in <a href="#sec-signature-rpk">RPK Signature Scheme</a>, including the developer signature and distributor signature.</p>
+            <p>This signature scheme includes the [=Provisioning Profile=] of the MiniApp and the developer's signature at the first stage (developer signing stage). At the second stage (distributor signing stage), a distributor MAY replace the developer's signature with the its own signature. Like the <a href="#sec-signature-rpk">RPK Signature Scheme</a>, this approach generates a Signing Block to be included into the package file, according to the <a href="sec-miniapp-digital-signature-requirements">Digital Signature Requirements</a>.</p>
```
Comment (espinr) [inline index.html:768]: The `#` is missing in the URL fragment:
`<a href="#sec-miniapp-digital-signature-requirements">Digital Signature Requirements</a>`

**Review by espinr — APPROVED** [review]
Thank you @zhangyongjing. It's clearer, and easier to understand. Only a minor correction (a link that was wrong).

File: index.html [hunk 2]
```diff
@@ -621,14 +619,11 @@ <h4>Signing Block Format</h4>
...
-                <ul>
-                    <li><code>0x01000101</code> Entire package signature, to guarantee the integrity of the MiniApp as a whole.</li>
-                    <li><code>0x01000201</code> File list signature</li>
-                </ul>
-
+                <p>The signatures are stored as <strong>ID-value pairs</strong> with specific identifiers that represent the types of the signature. The supported signatures, and associated IDs, for RPK packages are described in the following sub-clauses.</p>
+                
+                <p class="ednote" title="package-splitting"> There are cases where a MiniApp package is split into multiple sub-packages (by a distribution platform e.g. an app-store) for the user agent to download and execute it in a progressive fashion to improve the user experience. In such a case, additional signature types (thus IDs) would be needed to ensure the integrity of each sub-package as well as the whole. Details are ffs.</p>
```
Comment (xfq) [inline index.html:624]: What does "ffs" mean here?

Comment (zhangyongjing) [reply index.html]: Sorry, I update it to the full term - for further study.

**Pull request merged**